package.json

```diff
@@ -54,6 +54,7 @@
     "mocha-lcov-reporter": "^0.0.2",
     "swagger-test": "0.2.0",
     "url-template": "^2.0.6",
-    "nock": "^2.6.0"
+    "nock": "^2.6.0",
+    "restbase-mod-table-sqlite": "git+https://github.com/wikimedia/restbase-mod-table-sqlite#master"
```

Comment:
@d00rman There's one PR we are still pending there: https://github.com/wikimedia/restbase-mod-table-sqlite/pull/6 After that's resolved, we could release it